plyer/platforms/linux/screenshot.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+from plyer.facades import Screenshot
+from plyer.utils import whereis_exe
+
+
+class XwdScreenshot(Screenshot):
+    def __init__(self, file_path=None):
+        default_path = 'test.jpg'
+        super(XwdScreenshot, self).__init__(file_path or default_path)
+
+    def _take_shot(self):

```

Comment:
As I mentioned before, in my opinion, it's better to name methods without including the name of the object/parent/... in it. It looks bad and repetitive. Examples: `Window.window_size`, `Widget.widget_size` or `Screenshot.take_shot`. The "shot" part is already in the "Screenshot". Same applies for other `take_shot` methods in the rest of PRs.